android/expoview/src/main/java/versioned/host/exp/exponent/ExperiencePackagePicker.java

```diff
@@ -0,0 +1,58 @@
+package versioned.host.exp.exponent;
+
+import org.json.JSONObject;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import expo.core.interfaces.Package;
+import expo.modules.ads.admob.AdMobPackage;
+import expo.modules.analytics.segment.SegmentPackage;
+import expo.modules.barcodescanner.BarCodeScannerPackage;
+import expo.modules.camera.CameraPackage;
+import expo.modules.constants.ConstantsPackage;
+import expo.modules.contacts.ContactsPackage;
+import expo.modules.facedetector.FaceDetectorPackage;
+import expo.modules.filesystem.FileSystemPackage;
+import expo.modules.font.FontLoaderPackage;
+import expo.modules.gl.GLPackage;
+import expo.modules.google.signin.GoogleSignInPackage;
+import expo.modules.localauthentication.LocalAuthenticationPackage;
+import expo.modules.localization.LocalizationPackage;
+import expo.modules.location.LocationPackage;
+import expo.modules.medialibrary.MediaLibraryPackage;
+import expo.modules.payments.stripe.StripePackage;
+import expo.modules.permissions.PermissionsPackage;
+import expo.modules.print.PrintPackage;
+import expo.modules.sensors.SensorsPackage;
+import expo.modules.sms.SMSPackage;
+
+public class ExperiencePackagePicker {
+  private static final List<Package> EXPO_MODULES_PACKAGES = Arrays.<Package>asList(
+      new CameraPackage(),
+      new SensorsPackage(),
+      new FileSystemPackage(),
+      new FaceDetectorPackage(),
+      new ConstantsPackage(),
+      new GLPackage(),
+      new GoogleSignInPackage(),
+      new PermissionsPackage(),
+      new SMSPackage(),
+      new PrintPackage(),
+      new MediaLibraryPackage(),
+      new SegmentPackage(),
+      new FontLoaderPackage(),
+      new LocationPackage(),
+      new ContactsPackage(),
+      new BarCodeScannerPackage(),
+      new AdMobPackage(),
+      new StripePackage(),
+      new LocalAuthenticationPackage(),
+      new LocalizationPackage()
```

Comment:
let's make sure to add this to the new unimodule documentation

Reply:
Added [`Creating Universal Modules` guide](https://github.com/expo/expo/blob/%40sjchmiela/optional_modules_android/guides/Creating%20Universal%20Modules.md) 🙂